admin/class-openedx-woocommerce-plugin-admin.php

```diff
@@ -336,4 +336,121 @@ public function save_custom_product_fields( $post_id ) {
 		update_post_meta( $post_id, '_course_id', $course_id );
 		update_post_meta( $post_id, '_mode', $mode );
 	}
+
+	/**
+	 * This function is called when an order payment is completed.
+	 *
+	 * @param int $order_id Order id.
+	 *
+	 * @return void
+	 */
+	public function process_order_data( $order_id ) {
+
+		$order         = wc_get_order( $order_id );
+		$billing_email = $order->get_billing_email();
```

Comment:
The only variable that I can move inside the if statement is the billing_email because the order variable is used to get the status so if I put that variable inside the if statement it isn't gonna work.